mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/Person.kt

```diff
@@ -5,6 +5,9 @@ import ch.epfl.sdp.mobile.application.Profile.Color
 /** An interface representing a list of people you're following. */
 interface Person {
 
+  /** The the unique identifier of the Person. */
+  val uid: String
```

Comment:
You must not add a `uid` to `Person`. `Person` is a view interface, whereas the `uid` belongs to the model.